amplipi/streams/__init__.py

```diff
@@ -31,7 +31,7 @@
 from .internet_radio import InternetRadio
 from .rca import RCA
 from .airplay import AirPlay
-from .spotify import Spotify
+from .spotify_connect import SpotifyConnect
```

Comment:
I thought we had discussed just continuing to use our existing Spotify naming convention, and not calling this SpotifyConnect?

Reply:
The main thing I was trying to avoid was API breakage, this was fixed by using the stream_type=spotify. I left the class named spotify connect as a reminder of our intent

Reply:
If that seems too confusing it can be easily reverted.